### PR TITLE
created challenge object, passing it to search activity

### PR DESCRIPTION
**Pull request by Mila2010**

### CODE REVIEW

**Review by cito125 — CHANGES_REQUESTED** [review]
# Resolve merge conflicts
- Run a `git pull origin master`
- Followed by a`git checkout Mila`
- Then `git rebase master`
  - Resolve conflicts
- Finally, push your changes for a review

**Comment by Mila2010** [comment]
resolved conflicts and pushed to Mila again

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -15,21 +15,24 @@
...
         android:label="@string/app_name"
         android:supportsRtl="true"
+        android:largeHeap="true"
         android:theme="@style/AppTheme">
         <activity android:name=".login.SplashScreen" />
         <activity android:name=".login.StartActivity" >
 
         </activity>
-        <activity android:name=".homescreen.HomeScreenActivity"/>
+        <activity android:name=".homescreen.HomeScreenActivity">
```
Comment (cito125) [inline app/src/main/AndroidManifest.xml]: There's a merge conflict on where we're starting the app with the intent filter : (

File: app/src/main/java/com/example/andresarango/aughunt/challenge/Challenge.java [hunk 2]
```diff
@@ -2,48 +2,65 @@
...
+
 
 /**
  * Created by Millochka on 2/28/17.
  */
 
-public class Challenge<T> {
-    private T mChallenge;
-    private DAMLocation mLocation;
+public final class Challenge<T> {
```
Comment (cito125) [inline app/src/main/java/com/example/andresarango/aughunt/challenge/Challenge.java]: Should the entire class be final?

**Review by cito125 — CHANGES_REQUESTED** [review]

**Review by cito125 — APPROVED** [review]

**Comment by cito125** [comment]
I accidentally approved this, I apologize

**Pull request closed without merging**